torchvision/transforms/functional_tensor.py

```diff
@@ -50,15 +45,29 @@ def crop(img, top, left, height, width):
     return img[..., top:top + height, left:left + width]
 
 
+def rgb_to_grayscale(img):
+    """Convert the given RGB Image Tensor to Grayscale.
+
+    Args
+        img (Tensor): Image to be converted to Grayscale in the form [C, H, W].
+        Tensor: Grayscale image.
+
+    For RGB to Grayscale conversion, ITU-R 601-2 luma transform is performed which
```

Comment:
Can you move this before `Args:`, so that it's properly rendered during documentation generation?